library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -172,6 +214,7 @@ Http::FilterDataStatus PlatformBridgeFilter::onData(Buffer::Instance& data, bool
       data.drain(data.length());
       data.addBufferFragment(*Buffer::BridgeFragment::createBridgeFragment(result.data));
     }
+    iteration_state_ = IterationState::Ongoing;
```

Comment:
The absence of this assignment was a bug, right?

Reply:
It was, though not one we would have encountered yet. I am adding coverage for it.